d2l-custom-teacher-course-creation-admin.js

```diff
@@ -0,0 +1,53 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { LocalizeMixin } from '@brightspace-ui/core/mixins/localize-mixin.js';
+
+class CustomTeacherCourseCreation extends LocalizeMixin(LitElement) {
+
+	static get properties() {
+		return {
+			prop1: { type: String },
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: inline-block;
+			}
+			:host([hidden]) {
+				display: none;
+			}
+		`;
+	}
+
+	static async getLocalizeResources(langs) {
+		const langResources = {
+			'en': { 'myLangTerm': 'I am a localized string (in the admin page)!' }
+		};
+
+		for (let i = 0; i < langs.length; i++) {
+			if (langResources[langs[i]]) {
+				return {
+					language: langs[i],
+					resources: langResources[langs[i]]
+				};
+			}
+		}
+
+		return null;
+	}
+
+	constructor() {
+		super();
+
+		this.prop1 = 'custom-teacher-course-creation';
+	}
+
+	render() {
+		return html`
+			<h2>Hello ${this.prop1}!</h2>
+			<div>Localization Example: ${this.localize('myLangTerm')}</div>
+		`;
+	}
+}
+customElements.define('d2l-custom-teacher-course-creation', CustomTeacherCourseCreation);
```

Comment:
```suggestion
customElements.define('d2l-course-creation-admin', CustomTeacherCourseCreation);
```
Will need to be a different name or we'll run into issues when loading the component into the LMS.